### DISCUSSION TITLE
Can't download the video

### DISCUSSION THREAD

**Discussion by Giambozo** [aclap-dev/video-downloadhelper#218] [Help and Q&A]
### Your question

  I have this error when I try to download this video.
 https://ecomschool.ro/1.-introducere-curs-mentalitate
![image](https://github.com/aclap-dev/video-downloadhelper/assets/145464327/c981e2ca-2e69-447f-ae0b-77352193b1c2)

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: edge
lang: en-US
license: accepted
key (only 16 first characters): 10c9d7f62eb04c8f
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\lucyo",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
This is near identical to #208
